### PR TITLE
Adds label, index, value, and unit as properties of a draggable item.

### PR DESCRIPTION
**Pull request by eireland**
Adds a label, a value, and a unit label to each draggable item in authoring. 
Image size is also now authorable.

### CODE REVIEW

File: src/drag-and-drop/components/app.tsx
Comment (pjanik) [inline src/drag-and-drop/components/app.tsx:92]: What is index, itemValue and targetMatch? 
These values seem to be unused. I bet they're related to some upcoming features? Generally, I would avoiding merging that in this PR, as if the plans or schedule change, we might end up with some unused code that might be confusing. Or you might end up implementing that differently. 😉

**Review by pjanik — CHANGES_REQUESTED** [review]
Looks good. I've left one comment about unused properties. Otherwise looks good.

File: src/drag-and-drop/components/app.tsx
Comment (pjanik) [inline src/drag-and-drop/components/app.tsx:100]: Same here, seems to be unused.

File: src/drag-and-drop/components/app.tsx
Comment (pjanik) [inline src/drag-and-drop/components/app.tsx:77]: Why do we need now image width and height? It worked without it. I think it might be annoying for authors to provide pixel perfect dimensions for each image they use.

**Review by pjanik — CHANGES_REQUESTED** [review]
Ah, one more thing. Can we remove image width and height? Or, if it was specifically requested to be able to overwrite default dimensions, I'd make them optional.

File: src/drag-and-drop/components/app.tsx
Comment (eireland) [reply src/drag-and-drop/components/app.tsx:92]: Index, itemValue, and targetMatch are for upcoming features. They were specified to be in this story https://www.pivotaltracker.com/story/show/178247192
ItemValue and targetMatch will be exported to the graph question interactive.

File: src/drag-and-drop/components/app.tsx
Comment (eireland) [reply src/drag-and-drop/components/app.tsx:77]: It's an optionaly porperty if somebody needed to adjust their image to fit in the container. I ran into that problem when I was trying to find a water droplet to test with.

File: src/drag-and-drop/components/app.tsx
Comment (eireland) [reply src/drag-and-drop/components/app.tsx:100]: This was specified in the mockup here https://www.pivotaltracker.com/story/show/178218427. I'll take it out and put it in the PR for that story.

File: src/drag-and-drop/components/app.tsx
Comment (pjanik) [reply src/drag-and-drop/components/app.tsx:92]: Okay, so can we remove it from this PR and add them when they're actually used? 
Otherwise, we can end up with some unused confusing properties if the schedule or requirements change.

**Comment by pjanik** [comment]
@eireland, I wasn't sure if you're working on that or not. But before we merge, can we just remove unused properties and interfaces? That won't be any work to re-add them again when they're necessary.

File: src/drag-and-drop/components/types.ts
Comment (pjanik) [inline src/drag-and-drop/components/types.ts:13]: Unused props still here.

File: src/drag-and-drop/components/draggable-item.test.tsx
Comment (pjanik) [inline src/drag-and-drop/components/draggable-item.test.tsx:7]: Unused props still here.

**Review by pjanik — APPROVED** [review]
Thanks. There are a few more places with unused props. Feel free to clean it up after merging if that's easier for you, I don't want to hold this PR.

**Review by pjanik — APPROVED** [review]
Looks good, thanks

**Pull request merged**
